composer.json

```diff
@@ -54,6 +55,12 @@
             }
         }
     },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://nova.laravel.com"
+        }
+    ],
```

Comment:
Even though nova is not a direct dependency, this needs to be included in every repo so composer can find properly resolve the indirect dependency from tipoff/support.